### ISSUE TITLE
Alias plugin command descriptions

### ISSUE THREAD

**Issue by wflynny** [pubs/pubs#98]
I've added quite a few custom commands via the alias plugin (about as many as the default number of commands).  It would be helpful if we could specify a brief description of these commands that would be available to `pubs --help`, for example.

I've modified the `Alias` class in `pubs/plugs/alias/alias.py` so that I can include a small description in the alias definition in the conf file.  For now that looks like the following example:

    [[alias]]
    fadd = !pubs_fancyadd.py  <- interactive version of `pubs add`

where I've separated the alias definition from it's description with `<-`.  This could obviously be changed.

Is there any interest in adding functionality like this out of the box?  And are there any specific requirements that should be met while implementing it?

**Comment by omangin** [comment]
Thanks again! Having documentation of the alias function is indeed of interest! We have to check what makes the most sense for the configuration file syntax so that the parsing these does not interfere with the rest. Once that is fixed, it should be straightforward to implement. @benureau, any though?

**Comment by omangin** [comment]
So, @wflynny. Sorry for only coming back to it now. It seems that if we want to keep the compatibility with the `configobj`, there are basically two options here.

The first one is to use a bash comment starting with `#`. The `shlex` parser would be used to get the content of the comment that would be set as the alias documentation. The issue of this approach is that the comment needs to be included within a string value otherwise it is removed when the config is parsed. This means that in the following only the second comment will be available to `pubs`:
```
fadd = !pubs_fancyadd.py  # interactive version of `pubs add`
fadd = "!pubs_fancyadd.py  # interactive version of `pubs add`"
```

The second option is a little bit more cumbersome to write but more compatible with `configobj`, more consistent, and probably more straightforward to implement. It would consist in having the possibility of setting the alias as part of a subsection as follows:
```
[[alias]]
# The first two ones have no description
evince = open --with evince
count = !pubs list -k "$@" | wc -l
[[[fadd]]]
# This alias include a description
command = !pubs_fancyadd.py
description = "interactive version of `pubs add`"
```

So, if you still want to implement it, please do. I'm personally more in favor of the second option. In the future it could also be used for other arguments of aliases (such as for autocompletion).

**Comment by wflynny** [comment]
Yep that sounds easy enough.  Will probably get around to it over the weekend/holiday.

**Comment by omangin** [comment]
Closed by #104.
